sfge/src/main/java/com/salesforce/graph/symbols/PathScopeVisitor.java

```diff
@@ -1435,6 +1436,10 @@ private Optional<ApexValue<?>> handleApexValueMethod(MethodCallExpressionVertex
             // The method is being called on a chain
             // SObjectType sot = Schema.getGlobalDescribe().get('Account');
             apexValue = getReturnedValue(previous).orElse(null);
+        } else if (arrayInvocation != null) {
```

Comment:
This portion makes sure that a method call on a primitive or standard library instance array item gets the corresponding value returned.